src/core/terminology.md

```diff
@@ -5,19 +5,126 @@ permalink: /core/terminology
 # Terminology
 
 ## World
+The term `world` generally refers to the subject's field of view. Sometimes it is used
+in combination with other terms:
+
+- **World Camera**: Physical scene camera that captures the subject's field of view.
+- **World Process**: Application process responsible for processing the _world camera_
+    video stream and [_gaze mapping_](#gaze-positions).
+- **World Window**: The main Pupil Capture window; previews the result of the _world process_
+- **World Coordinate System**: Refers to the [_Coordinate System_](#coordinate-system) of the _world camera_.
 
 ## Eye
+The term `eye` is always used in combination with a different term that sppecifies the context:
+
+- **Eye ID**: `0` or `1`; identifies the right and left eyes.
+- **Eye Camera**: Physical camera capturing the subject's eye. There can be one or two
+    eye cameras, depending on your setup.
+- **Eye Process**: Application process responsible for processing the _eye camera_ video
+    stream and [_pupil detection_](#pupil-positions).
+- **Eye Window**: Previews the result of the _eye process_.
+- **Eye Coordinate System**: Refers to the [_Coordinate System_](#coordinate-system) of the _eye camera_.
+- **Eye Model**: Result of the _3d pupil detection_.
+
+## Pupil Positions
+Alternatively: _Pupil Data_. Output of the _pupil detection_. Location of the pupil
+within the _eye coordinate system_.
 
-## Pupil
+- **Pupil Detection**: Process of detecting the pupil within the _eye camera_ video stream.
+- **2d Pupil Detection**: Attempts to find a 2d ellipse that fits the pupil within a given _eye_ image.
+    [Reference paper](https://arxiv.org/pdf/1405.0006.pdf).
+- **3d Pupil Detection**: Uses a series of 2D ellipses to fit a 3d _eye model_.
+    [Reference paper](https://www.researchgate.net/profile/Lech_Swirski/publication/264658852_A_fully-automatic_temporal_approach_to_single_camera_glint-free_3D_eye_model_fitting/links/53ea3dbf0cf28f342f418dfe/A-fully-automatic-temporal-approach-to-single-camera-glint-free-3D-eye-model-fitting.pdf).
 
-## Gaze
+## Gaze Positions
+Alternatively: _Gaze Data_. Output of the _gaze estimation_. Location of the subject's
+gaze within the _world coordinate system_.
+
+- **Gaze Estimation**: Process of mapping a Pupil Position from the _eye coordinate system_
+    to the _world coordinate system_.
 
 ## Confidence
 
+Quality assessment of the _pupil detection_ for a given _eye_ image.
+- `0.0` means that the pupil could not be detected.
+- `1.0` highest possible value; the pupil was detected with very high certainty.
+
+_Gaze data_ inherits the _confidence_ value from the _pupil data_ it was based on.
+
 ## Calibration
 
```

Comment:
This can be explained in a simpler way for general audience IMO.

Reply:
I am failing at finding an other abstraction level, that is not just `/core/#_4-calibration`... Did you have a specific simplification in mind?